pkg/v1/tkg/test/tkgctl/tkgs/tkgs_workload_cluster_test.go

```diff
@@ -131,38 +131,112 @@ var _ = Describe("TKGS - Create workload cluster use cases", func() {
+				mngClient, mngDynamicClient, mngAggregatedAPIResourcesClient, mngDiscoveryClient, err := shared.GetClients(context.Background(), mngTempFilePath)
+				Expect(err).NotTo(HaveOccurred())
+
+				By(fmt.Sprintf("Generating credentials for workload cluster %q", e2eConfig.WorkloadClusterOptions.ClusterName))
+				wlcKubeConfigFileName := e2eConfig.WorkloadClusterOptions.ClusterName + ".kubeconfig"
+				wlcTempFilePath := filepath.Join(os.TempDir(), wlcKubeConfigFileName)
+				err = tkgctlClient.GetCredentials(tkgctl.GetWorkloadClusterCredentialsOptions{
+					ClusterName: clusterName,
+					Namespace:   namespace,
+					ExportFile:  wlcTempFilePath,
+				})
+				Expect(err).To(BeNil())
+
+				By(fmt.Sprintf("Get k8s client for workload cluster %q", clusterName))
+				wlcClient, _, _, _, err := shared.GetClients(context.Background(), wlcTempFilePath)
+				Expect(err).NotTo(HaveOccurred())
+
+				By(fmt.Sprintf("Verify addon packages on management cluster %q matches clusterBootstrap info on management cluster %q", clusterName, clusterName))
+				err = shared.CheckClusterCB(context.Background(), mngClient, wlcClient, clusterName, namespace, "", "", e2eConfig.InfrastructureName, true)
+				Expect(err).To(BeNil())
+
+				By(fmt.Sprintf("Verify addon packages on workload cluster %q matches clusterBootstrap info on management cluster %q", e2eConfig.WorkloadClusterOptions.ClusterName, clusterName))
+				err = shared.CheckClusterCB(context.Background(), mngClient, wlcClient, clusterName, namespace, clusterName, namespace, e2eConfig.InfrastructureName, false)
+				Expect(err).To(BeNil())
+
+				By(fmt.Sprintf("Get management cluster resources created by addons-manager for workload cluster %q on management cluster %q", e2eConfig.WorkloadClusterOptions.ClusterName, clusterName))
+				clusterResources, err = shared.GetManagementClusterResources(context.Background(), mngClient, mngDynamicClient, mngAggregatedAPIResourcesClient, mngDiscoveryClient, namespace, clusterName, e2eConfig.InfrastructureName)
+				Expect(err).NotTo(HaveOccurred())
+				Expect(clusterResources).ToNot(BeEmpty())
+				*/
+
+				clusterClient := framework.GetClusterclient(e2eConfig.TKGSKubeconfigPath, e2eConfig.TKGSKubeconfigContext)
+				config := &cniv1alpha1.AntreaConfig{}
+				err := clusterClient.GetResource(config, clusterName, namespace, nil, nil)
+				Expect(err).NotTo(BeNil())
+				Expect(config.Spec.Antrea.AntreaConfigDataValue.FeatureGates.AntreaTraceflow).Should(Equal(false))
+
+				deleteClusterClassBasedCluster(tkgctlClient, deleteClusterOptions, clusterName, namespace)
+
 			})
 		})
 	})
 })
 
 // createClusterClassBasedClusterTest creates and deletes (if created successfully) workload cluster
-func createClusterClassBasedClusterTest(tkgctlClient tkgctl.TKGClient, deleteClusterOptions tkgctl.DeleteClustersOptions, cliFlag bool, clusterName, namespace string) {
+func createClusterClassBasedCluster(tkgctlClient tkgctl.TKGClient, cliFlag bool, clusterName, namespace string) {
```

Comment:
hmmm. I absolutely understand why you would want to rename function, but if it is just cosmetic, best no to include changes like this in a patch.... make the patch longer, and the compile code ends up exactly the same. 
If however you are refactoring, i.e. create a function createClusterClassBasedCluster and call it from several places. ok.